### PR TITLE
Remove Freight role

### PR DESCRIPTION
**Pull request by SilasPeters**
Since Magnificent Sloth no longer requires setup through freight but through a GitHub Action, the role is no longer used. Meanwhile, it can't be deployed on Ubuntu 22, as the package is not available for that Ubuntu version.

Thus, this PR aims to remove the freight role completely.

### CODE REVIEW

**Comment by SilasPeters** [comment]
Note that this is still untested

File: ansible/main.yml [hunk 1]
```diff
@@ -40,8 +40,6 @@
       tags: "websites"
     - role: "digidecs"
       tags: "digidecs"
-    # - role: "freight"  Commented out, as this seems to be unused and is incompatible with Ubuntu 22.04
```
Comment (HugoPeters1024) [inline ansible/main.yml:43]: this comment can now go

**Review by HugoPeters1024 — APPROVED** [review]
LGTM modulo 1 leftover comment. Don't forget to deploy this to the current production server first. We don't want the migration to include changes to the playbook itself as well.

File: ansible/main.yml (hunk 1 above)
Comment (SilasPeters) [reply ansible/main.yml:43]: Silly me, only removed the tags line.

Comment (SilasPeters) [reply ansible/main.yml:43]: Wait, I am literally making the comment go away in the latest commit. Perhaps I am misunderstanding you, but line 43 and 44 are now gone.

Comment (HugoPeters1024) [reply ansible/main.yml:43]: wow you're right, brainfart

**Pull request merged**